### PR TITLE
Small fixes for 6 modules

### PR DESCRIPTION
**Pull request by asankov**

### CODE REVIEW

File: tcms/testcases/forms.py [hunk 1]
```diff
@@ -144,14 +145,14 @@ def clean_is_automated(self):
     def clean_script(self):
         if self.script_val == '':
             return u''
-        elif self.script_val:
+        if self.script_val:
```
Comment (atodorov) [inline tcms/testcases/forms.py]:
```
if self.script_val:
    return self.cleaned_data['script']

return ''
```

same for notes as well.

File: tcms/testcases/forms.py [hunk 2]
```diff
@@ -338,21 +339,25 @@ class QuickSearchCaseForm(forms.Form):
...
+            return case_ids
+        except ValueError:
+            raise forms.ValidationError(VALIDATION_ERROR_MESSAGE)
+
+
+VALIDATION_ERROR_MESSAGE = 'Please input valid case id(s). ' \
+          'use comma to split more than one ' \
+          'case id. e.g. "111, 222"'
 
 # =========== Mist Forms ==============
```
Comment (atodorov) [inline tcms/testcases/forms.py]: Remove this comment and update whitespace.

File: tcms/testcases/forms.py [hunk 3]
```diff
@@ -338,21 +339,25 @@ class QuickSearchCaseForm(forms.Form):
...
+        try:
+            case_ids = []
+            for case_id in case_id_set.split(','):
+                case_ids.append(int(case_id))
+            return case_ids
+        except ValueError:
+            raise forms.ValidationError(VALIDATION_ERROR_MESSAGE)
+
+
+VALIDATION_ERROR_MESSAGE = 'Please input valid case id(s). ' \
```
Comment (atodorov) [inline tcms/testcases/forms.py]: Use `ugettext_lazy()` to mark this as translatable and also update translation strings by `makemessages`

File: tcms/testcases/forms.py [hunk 4]
```diff
@@ -338,21 +339,25 @@ class QuickSearchCaseForm(forms.Form):
...
-                                            'use comma to split more than one '
-                                            'case id. e.g. "111, 222"')
-        else:
-            raise forms.ValidationError('Please input valid case id(s). '
-                                        'use comma to split more than one '
-                                        'case id. e.g. "111, 222"')
 
+        if not case_id_set:
+            raise forms.ValidationError(VALIDATION_ERROR_MESSAGE)
+        try:
```
Comment (atodorov) [inline tcms/testcases/forms.py]: Add 1 empty line between the two statements for readability.

File: tcms/testcases/forms.py (hunk 3 above)
Comment (atodorov) [reply tcms/testcases/forms.py]: Also it is better to define constants before the function that uses them, or even better at the top of the module so thay can easily be found.

Comment (asankov) [reply tcms/testcases/forms.py]: Done, althought a lot of lines appears to have changes in the translations files. Please verify it all is ok, or I will drop the commit

File: tcms/testcases/forms.py (hunk 1 above)
Comment (asankov) [reply tcms/testcases/forms.py]: Done

File: tcms/testcases/forms.py (hunk 3 above)
Comment (asankov) [reply tcms/testcases/forms.py]: NVM, there were conflicts, which were resolved. Should be fine now

File: tcms/testcases/forms.py [hunk 5]
```diff
@@ -144,16 +150,18 @@ def clean_is_automated(self):
     def clean_script(self):
         if self.script_val == '':
             return u''
```
Comment (atodorov) [inline tcms/testcases/forms.py]: This is now redundant.

File: tcms/testcases/forms.py [hunk 6]
```diff
@@ -144,16 +150,18 @@ def clean_is_automated(self):
...
-        elif self.script_val:
+        if self.script_val:
             return self.cleaned_data['script']
-        return None
+
+        return ''
 
     def clean_notes(self):
         if self.notes_val == '':
             return u''
```
Comment (atodorov) [inline tcms/testcases/forms.py]: redundant

**Pull request merged**